R/item_rename_files.R

```diff
@@ -0,0 +1,42 @@
+#' @title Rename item attached files
+#' 
+#' @description Renames files attached to an SB item. 
+#'
+#' 
+#' @template manipulate_item
+#' @param names List of names of files to rename
+#' @param new_names List of new file names to use
+#' 
+#' @examples 
+#' \dontrun{
+#' 
+#' names = c('file1.txt', 'file2.txt')
+#' new_names = c('newname1.txt', 'newname2.txt')
+#' 
+#' item_rename_files('sbid', names, new_names)
+#' 
+#' }
+#' 
+#' @export
+item_rename_files = function(sb_id, names, new_names, ..., session=current_session()){
```

Comment:
hey, cool! don't have time to inspect more now, but excited that you wrote the function